resources/cncf/README.md

```diff
@@ -6,78 +6,78 @@
 
 ## 基本介绍
 
-[CNCF](https://www.cncf.io/)，全称Cloud Native Computing Foundation（云原生计算基金会），口号是**坚持和整合开源技术来让编排容器作为微服务架构的一部分**。
+[CNCF](https://www.cncf.io/)，全称 Cloud Native Computing Foundation（云原生计算基金会），口号是 ** 坚持和整合开源技术来让编排容器作为微服务架构的一部分 **。
```

Comment:
Space after `**` breaks the syntax I think.

Reply:
Fixed, it is a bug.

Reply:
Maybe an upstream issue?

Reply:
Sure, it is a bug from the tool I used. I plan to implement a well maintained VS Code plugin while I do not think the wheel will be finished soon.